frameit/lib/frameit/device_types.rb

```diff
@@ -1,11 +1,13 @@
 module Frameit
   module Color
-    BLACK = "SpaceGray"
-    SILVER = "Slvr"
+    BLACK = "Space Gray"
```

Comment:
Just a quick note: the iPhone 7 has only 'Black' listed as color, there is no more Space Gray